src/components/widget/widget.component.ts

```diff
@@ -81,6 +80,13 @@ const WidgetComponent: ng.IComponentOptions = {
         chartRequest.query = filters
           .map((f) => '(' + f + ':' + queryFilters[f].map((qp) => this.AnalyticsService.buildQueryParam(qp, f)).join(' OR ') + ')')
           .join(' AND ');
+
+        if (this.widget.chart.request.query) {
+          if (chartRequest.query) {
+            chartRequest.query = `${chartRequest.query} AND `;
+          }
+          chartRequest.query = `${chartRequest.query} ${this.widget.chart.request.query}`;
```

Comment:
The space is useless here.
If a query already exists, a space is added in the 'if' statement.
If not, no need to add a space.
```suggestion
          chartRequest.query = `${chartRequest.query}${this.widget.chart.request.query}`;
```